### PR TITLE
Created dashboard component

### PR DESCRIPTION
**Pull request by moustf**
- Configured the instance of react-router for the project.
- Configured the instance of the i18next localization library for the project. 
- Created the class component.
- Created the dashboard component.
- Added a dummy stats component to illustrate how i18next logic works for the team.
---
Screenshot from 2022-10-21 22-54-11

### CODE REVIEW

File: client/src/components/Class/ClassDashboard/ClassDashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    color: "#B8C5D3",
+  },
+  layout: { padding: "0 24px 24px" },
+  breadcrumb: { margin: "16px 0" },
+  layoutContent: {
+    padding: 24,
+    margin: 0,
+    minHeight: 280,
+  },
+};
```
Comment (mahmoudhammad309) [inline client/src/components/Class/ClassDashboard/ClassDashboard.tsx:80]: to keep your code clean you should move the style to an external CSS file

File: client/src/components/StatsDummy/Dummy.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+const StatsDummy: React.FC = () => {
+  const { t }: any = useTranslation();
```
Comment (mahmoudhammad309) [inline client/src/components/StatsDummy/Dummy.tsx:5]: what's "t"?

File: client/src/components/Class/ClassDashboard/ClassDashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  CarryOutOutlined,
+  BellFilled,
+  MenuOutlined,
+} from "@ant-design/icons";
+import React from "react";
+import { Outlet, NavLink } from "react-router-dom";
+import type { MenuProps } from "antd";
+import { Breadcrumb, Layout, Menu } from "antd";
+import Avatar from "../../../assets/avatar.png";
+import Logo from "../../../assets/Logo.png";
```
Comment (mahmoudhammad309) [inline client/src/components/Class/ClassDashboard/ClassDashboard.tsx:16]: our project name is "GeekSchool"

File: client/src/components/Class/ClassDashboard/ClassDashboard.css [hunk 4]
```diff
@@ -0,0 +1,20 @@
+.class-dashboard {
+  height: 100vh;
+}
+
+#components-layout-demo-top-side-2 .logo {
+  float: left;
```
Comment (mahmoudhammad309) [inline client/src/components/Class/ClassDashboard/ClassDashboard.css:6]: float?

**Review by mahmoudhammad309 — CHANGES_REQUESTED** [review]

File: client/src/components/Class/ClassDashboard/ClassDashboard.css (hunk 4 above)
Comment (moustf) [reply client/src/components/Class/ClassDashboard/ClassDashboard.css:6]: Antd styles, not mine.

File: client/src/components/Class/ClassDashboard/ClassDashboard.tsx (hunk 3 above)
Comment (moustf) [reply client/src/components/Class/ClassDashboard/ClassDashboard.tsx:16]: It's temporary, not the official one.

File: client/src/components/StatsDummy/Dummy.tsx (hunk 2 above)
Comment (moustf) [reply client/src/components/StatsDummy/Dummy.tsx:5]: The function which is responsible for localization -i18next library-

File: client/src/components/Class/ClassDashboard/ClassDashboard.tsx (hunk 1 above)
Comment (moustf) [reply client/src/components/Class/ClassDashboard/ClassDashboard.tsx:80]: It's valid both ways, but I'll move them if you put your'all styles in a CSS file.

**Review by Raghad-Mezied — APPROVED** [review]

File: client/src/App.tsx [hunk 5]
```diff
@@ -1,8 +1,61 @@
...
+        path: "/class/questions",
+        element: <h1>Questions</h1>,
+      },
+      {
+        path: "/class/feedback",
+        element: <h1>Feedback</h1>,
+      },
+      {
+        path: "/class/recommended",
+        element: <h1>Recommended</h1>,
```
[the remark this answers is not in the document]
Comment (moustf) [reply client/src/App.tsx:48]: We tried before if you remember, but it didn't work!

**Review by Aaya-Elsharief — APPROVED** [review]

**Pull request merged**